src/register.coffee

```diff
@@ -104,6 +104,9 @@ module.exports = (options={}) ->
 
     replaceHandler = (extension) ->
         origCoffeeHandler = require.extensions[extension]
+        # just expose needed API in some hideous way....
+        fs.instrumentFile ||= (fileName) ->
```

Comment:
@jwalton you're absolutely correct. :) At that time, I was just trying the stuff to work.. So, I abused the `fs`... :p